src/main/resources/db/migration/common/V1_280__add_regions.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO region (reg_id, region_name) VALUES(1, 'London');
+INSERT INTO region (reg_id, region_name) VALUES(2, 'South East');
+INSERT INTO region (reg_id, region_name) VALUES(3, 'South West');
+INSERT INTO region (reg_id, region_name) VALUES(4, 'Midlands');
+INSERT INTO region (reg_id, region_name) VALUES(5, 'North East');
+INSERT INTO region (reg_id, region_name) VALUES(6, 'North West');
+INSERT INTO region (reg_id, region_name) VALUES(7, 'Wales');
```

Comment:
on staging DB we already have the following:
-11  Wales
-10  London
I wonder whether we want to do some cleanup there

Reply:
Hey Mario. Yes. Good point, the data in staging could do with a cleanup, I assume this would be on a separate ticket and would require a fair amount of collaboration with the testers.